logging/logging.go

```diff
@@ -274,6 +275,7 @@ func (c *Client) Logger(logID string, opts ...LoggerOption) *Logger {
 		populateSourceLocation: DoNotPopulateSourceLocation,
 		partialSuccess:         false,
 		redirectOutputWriter:   nil,
+		minimalLoggedSeverity:  Default,
```

Comment:
This should be mentioned with the [other LoggerOptions in the README](https://github.com/googleapis/google-cloud-go/blob/56f4cdd066cc9eaeece2c6fb466d58c3e7c41563/logging/README.md#logger-configuration-options)

Reply:
Sure. I will add this to README.

Reply:
Done.